core/src/main/java/org/kiwix/kiwixmobile/core/extensions/ImageViewExtensions.kt

```diff
@@ -25,13 +25,11 @@ import androidx.core.widget.ImageViewCompat
 import org.kiwix.kiwixmobile.core.downloader.model.Base64String
 
 fun ImageView.setBitmap(base64String: Base64String) {
-  if (tag != base64String) {
-    base64String.toBitmap()
-      ?.let {
-        setImageBitmap(it)
-        tag = base64String
-      }
-  }
+  base64String.toBitmap()
+    ?.let {
+      setImageBitmap(it)
+      tag = base64String
```

Comment:
we don't need to set the tag either

Reply:
Also here is a great video on [debugging](https://www.youtube.com/watch?v=rjlhSDhFwzM) if you get the free time.

Reply:
Thanks!! I'll definitely look into it.